client/src/services/Stripe.ts

```diff
@@ -0,0 +1,3 @@
+import { loadStripe } from "@stripe/stripe-js"
+
+export const stripePromise = loadStripe(import.meta.env.VITE_ENV_STRIPE_API_KEY)
```

Comment:
Can we rename this to `VITE_ENV_STRIPE_PUBLISHABLE_KEY` instead?

Reply:
done